tests/hash_collision.c

```diff
@@ -18,6 +18,7 @@ main (int argc, char **argv)
   char *table = "tests/tables/empty.ctb";
   char rule[18];
 
+  lou_getTable(table);
```

Comment:
Good question.

In the general case, `lou_compileString` now adds to the current (last compiled) display table. When I did this change I was convinced this was needed but now I'm not so sure anymore.

Anyway, because in this specific case there is no current display table, and so a new one is automatically created with the same name as the translation table, it doesn't matter whether this line is present or not, but I added it anyway for clarity, because you will need to do this now if you want to create a new display table with `lou_compileString` (when one may already exist).

But I'm thinking about reverting the behavior of `lou_compileString` and creating a new version of this function for use in lou_checkyaml.c if needed.



1. It would otherwise be impossible to create a display table with this function. Note however that it is still problematic to do so, and I am not actually using this feature, see lou_checkyaml.c. Also I could have created a new (internal) version of this function to add lines to a display table.

2. But